docs/plugins/loadext.rst

```diff
@@ -0,0 +1,53 @@
+Load Extension Plugin
+=====================
+
+Beets uses an SQLite database to store and query library information, which
+has support for extensions to extend its functionality.
+
+One of the primary uses of this within beets is with the `"ICU" extension`_,
+which adds support for case insensitive querying of non-ASCII characters.
+
+.. _"ICU" extension: https://www.sqlite.org/src/dir?ci=7461d2e120f21493&name=ext/icu
+
+Configuration
+-------------
+
+To configure the plugin, make a ``loadext`` section in your configuration
+file. The section must consist of a list of paths to extensions to load.
```

Comment:
I recommend putting the example right here, instead of later down in the last section. You can introduce it by saying something like "The section should look like this:" and list the YAML right there. This makes the introduction mostly self-contained.